training/src/main/java/wolox/training/models/Users.java

```diff
@@ -61,38 +63,19 @@ public Users(String username, String name, LocalDate birthdate, String password)
         this.password = password;
     }
 
-    public String getPassword() {
-        return password;
-    }
-
-    public void setPassword(String password) {
-        this.password = password;
```

Comment:
I'm sorry I've missed it before: as you should always encode the password, the encoding should be done here instead of doing it in the controller as you are doing now. Considering it was my mistake to not tell you this before, it is not necessary for you to update it, but if someday you want to use your training to check how things should be done, don't forget about this.